### ISSUE TITLE
support for chrome extension IMEs

### ISSUE THREAD

**Issue by booperlv** [Miodec/monkeytype#663]
I am a Chrome OS user, and I genuinely enjoy typing on this website. However, I am an alternate keyboard layout user, and because of the limits on Chrome OS, the more custom ones can only be used using an extension. Actually, monkeytype does support the native Chrome OS layouts such as Colemak and Dvorak, but extensions are not detected by the default layer. I understand that there are multiple on site options to tackle the issue, but I am currently trying to create my own/modify a few layouts according to my nitpicks about them.

Basically, please add support for recognizing Chrome Extension IMEs as an actual layout. I found that using the layout would work in parts of the website such as the search bar on esc press, however - it defaults to qwerty automatically in the actual text box.
Thanks in advance!

**Comment by Miodec** [comment]
Did you try disabling layout override? (Setting it to default)

**Comment by booperlv** [comment]
Sorry for the late reply, layout override is already disabled, but it still defaults to qwerty in the actual typing test

**Comment by Miodec** [comment]
Well, it looks like chrome extensions are applied just before editing the input fields, without changing the key events. Since MT looks at keyevents, not input events - this might not be possible at the moment without rewriting the input system.

**Comment by booperlv** [comment]
Got it, thanks for the fast replies though! (Faster than the how often I check github)
AFAIK, It does change the key event values after the event happens - unlike the native ones which completely remap the key events. I probably should have checked the source code first. Thanks for the responses again!
